### DISCUSSION TITLE
WebRtc player

### DISCUSSION THREAD

**Discussion by aimaicai** [bluenviron/mediamtx#1316] [Questions]
Hi, thanks for the amazing work. I'm trying to view the stream from webrtc, but i I connect throught browser to the server address, at the port 8889, it starts a video player, but nothing is playing. How can I do? Must I use a specific player? Sorry, but I am not very clever on it. Thanks

Roberto

**Comment by delphikit** [comment]
You can try to use an iframe. Set the source (src) to the webrtc url (e.g. http://yourserver:8889/yourstream).

[a comment by an automated account is not shown]
